src/saveload/saveload.h

```diff
@@ -325,8 +326,10 @@ enum SaveLoadVersion : uint16 {
 	SLV_GS_INDUSTRY_CONTROL,                ///< 287  PR#7912 and PR#8115 GS industry control.
 	SLV_VEH_MOTION_COUNTER,                 ///< 288  PR#8591 Desync safe motion counter
 	SLV_INDUSTRY_TEXT,                      ///< 289  PR#8576 v1.11.0-RC1  Additional GS text for industries.
+
```

Comment:
What is this line intentionally left blank for? :p

Reply:
my OCD at work! Till recently, all these blobs were grouped per 5, at the (% 5 ==0). But recently that got a bit distorted. So I restored it in all its glory!

Reply:
Haha, that was not visible from the diff :)